js/ui/lookup.js

```diff
@@ -623,6 +629,93 @@ var Lookup = DropDownList.inherit({
                 options: {
                     useInkRipple: true
                 }
+            },
+            {
+                device: function() {
+                    return themes.isMaterial();
+                },
+                options: {
+
+                    /**
+                    * @name dxLookupOptions_usePopover
+                    * @publicName usePopover
+                    * @type boolean
+                    * @default false @for Material
+                    */
+                    usePopover: false,
+
+                    /**
+                    * @name dxLookupOptions_closeOnOutsideClick
+                    * @publicName closeOnOutsideClick
+                    * @type boolean|function
+                    * @default true @for Material
+                    * @type_function_return boolean
+                    */
+
+                    closeOnOutsideClick: true,
+
+                    /**
+                     * @name dxLookupOptions_popupWidth
+                     * @publicName popupWidth
+                     * @type number|string|function
+                     * @default undefined @for Material
```

Comment:
But default value is `(function() { return this._$element.outerWidth(); }).bind(this)`